Crash when trying to create a new world

**The crash has been introduced in 1.2.0**
As soon as I click on the "Create new world" button, the entire minecraft client crashes (It's seems to be caused by twelvemonkeys imageio).
For some reasons that's not reproducible in the dev enviroment

```
[16:24:24] [Render thread/ERROR]: Reported exception thrown!
net.minecraft.class_148: mouseClicked event handler
	at knot/net.minecraft.class_312.method_1601(class_312.java:129) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_312.method_22686(class_312.java:226) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.execute(class_1255.java:110) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_312.method_22684(class_312.java:226) ~[client-intermediary.jar:?]
	at knot/org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43) ~[lwjgl-glfw-3.3.3.jar:?]
	at knot/org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.3.3.jar:?]
	at knot/org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3509) ~[lwjgl-glfw-3.3.3.jar:?]
	at knot/com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:164) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_310.method_1523(class_310.java:1375) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_310.method_1514(class_310.java:936) [client-intermediary.jar:?]
	at knot/net.minecraft.client.main.Main.main(Main.java:265) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:506) [fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72) [fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.17.0.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:102) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) [NewLaunch.jar:?]
Caused by: java.lang.NoClassDefFoundError: com/twelvemonkeys/imageio/spi/ImageReaderSpiBase
	at java.base/java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027) ~[?:?]
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:160) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:355) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.17.0.jar:?]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
	at java.base/java.lang.Class.forName0(Native Method) ~[?:?]
	at java.base/java.lang.Class.forName(Class.java:534) ~[?:?]
	at java.base/java.lang.Class.forName(Class.java:513) ~[?:?]
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.nextProviderClass(ServiceLoader.java:1217) ~[?:?]
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1228) ~[?:?]
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1273) ~[?:?]
	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1309) ~[?:?]
	at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1393) ~[?:?]
	at java.desktop/javax.imageio.spi.IIORegistry.registerApplicationClasspathSpis(IIORegistry.java:179) ~[?:?]
	at java.desktop/javax.imageio.spi.IIORegistry.<init>(IIORegistry.java:107) ~[?:?]
	at java.desktop/javax.imageio.spi.IIORegistry.getDefaultInstance(IIORegistry.java:128) ~[?:?]
	at java.desktop/javax.imageio.ImageIO.<clinit>(ImageIO.java:64) ~[?:?]
	at knot/com.thedeathlycow.novoatlas.registry.MultiFileTypeToIdConverter.builtinImages(MultiFileTypeToIdConverter.java:27) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/com.thedeathlycow.novoatlas.registry.MultiFileTypeToIdConverter.imageRegistry(MultiFileTypeToIdConverter.java:35) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/com.thedeathlycow.novoatlas.registry.ImageManager.reload(ImageManager.java:37) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/com.thedeathlycow.novoatlas.fabric.MapImageLoader.method_14491(MapImageLoader.java:19) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/net.minecraft.class_4013.method_29490(class_4013.java:16) ~[client-intermediary.jar:?]
	at java.base/java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:787) ~[?:?]
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	at knot/net.minecraft.class_4014.method_67574(class_4014.java:60) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_18859(class_1255.java:164) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_4093.method_18859(class_4093.java:23) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_16075(class_1255.java:138) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_18857(class_1255.java:147) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_525.method_64244(class_525.java:373) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_525.method_64368(class_525.java:321) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_525.method_31130(class_525.java:308) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_526.method_19944(class_526.java:54) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_4185.method_25306(class_4185.java:96) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_4264.method_25348(class_4264.java:43) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_339.method_25402(class_339.java:141) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_4069.method_25402(class_4069.java:43) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_312.mixinextras$bridge$method_25402$38(class_312.java) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_312.wrapOperation$zoa000$fabric-screen-api-v1$invokeMouseClickedEvents(class_312.java:545) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_312.method_1601(class_312.java:120) ~[client-intermediary.jar:?]
	... 16 more
Caused by: java.lang.ClassNotFoundException: com.twelvemonkeys.imageio.spi.ImageReaderSpiBase
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.17.0.jar:?]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
	at java.base/java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027) ~[?:?]
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:160) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:355) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.17.0.jar:?]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
	at java.base/java.lang.Class.forName0(Native Method) ~[?:?]
	at java.base/java.lang.Class.forName(Class.java:534) ~[?:?]
	at java.base/java.lang.Class.forName(Class.java:513) ~[?:?]
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.nextProviderClass(ServiceLoader.java:1217) ~[?:?]
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1228) ~[?:?]
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1273) ~[?:?]
	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1309) ~[?:?]
	at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1393) ~[?:?]
	at java.desktop/javax.imageio.spi.IIORegistry.registerApplicationClasspathSpis(IIORegistry.java:179) ~[?:?]
	at java.desktop/javax.imageio.spi.IIORegistry.<init>(IIORegistry.java:107) ~[?:?]
	at java.desktop/javax.imageio.spi.IIORegistry.getDefaultInstance(IIORegistry.java:128) ~[?:?]
	at java.desktop/javax.imageio.ImageIO.<clinit>(ImageIO.java:64) ~[?:?]
	at knot/com.thedeathlycow.novoatlas.registry.MultiFileTypeToIdConverter.builtinImages(MultiFileTypeToIdConverter.java:27) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/com.thedeathlycow.novoatlas.registry.MultiFileTypeToIdConverter.imageRegistry(MultiFileTypeToIdConverter.java:35) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/com.thedeathlycow.novoatlas.registry.ImageManager.reload(ImageManager.java:37) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/com.thedeathlycow.novoatlas.fabric.MapImageLoader.method_14491(MapImageLoader.java:19) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/net.minecraft.class_4013.method_29490(class_4013.java:16) ~[client-intermediary.jar:?]
	at java.base/java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:787) ~[?:?]
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	at knot/net.minecraft.class_4014.method_67574(class_4014.java:60) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_18859(class_1255.java:164) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_4093.method_18859(class_4093.java:23) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_16075(class_1255.java:138) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_18857(class_1255.java:147) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_525.method_64244(class_525.java:373) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_525.method_64368(class_525.java:321) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_525.method_31130(class_525.java:308) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_526.method_19944(class_526.java:54) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_4185.method_25306(class_4185.java:96) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_4264.method_25348(class_4264.java:43) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_339.method_25402(class_339.java:141) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_4069.method_25402(class_4069.java:43) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_312.mixinextras$bridge$method_25402$38(class_312.java) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_312.wrapOperation$zoa000$fabric-screen-api-v1$invokeMouseClickedEvents(class_312.java:545) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_312.method_1601(class_312.java:120) ~[client-intermediary.jar:?]
	... 16 more
---- Minecraft Crash Report ----
// Ouch. That hurt :(

Time: 2025-07-31 16:24:24
Description: mouseClicked event handler

java.lang.NoClassDefFoundError: com/twelvemonkeys/imageio/spi/ImageReaderSpiBase
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:160)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:355)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:534)
	at java.base/java.lang.Class.forName(Class.java:513)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.nextProviderClass(ServiceLoader.java:1217)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1228)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1273)
	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1309)
	at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1393)
	at java.desktop/javax.imageio.spi.IIORegistry.registerApplicationClasspathSpis(IIORegistry.java:179)
	at java.desktop/javax.imageio.spi.IIORegistry.<init>(IIORegistry.java:107)
	at java.desktop/javax.imageio.spi.IIORegistry.getDefaultInstance(IIORegistry.java:128)
	at java.desktop/javax.imageio.ImageIO.<clinit>(ImageIO.java:64)
	at knot//com.thedeathlycow.novoatlas.registry.MultiFileTypeToIdConverter.builtinImages(MultiFileTypeToIdConverter.java:27)
	at knot//com.thedeathlycow.novoatlas.registry.MultiFileTypeToIdConverter.imageRegistry(MultiFileTypeToIdConverter.java:35)
	at knot//com.thedeathlycow.novoatlas.registry.ImageManager.reload(ImageManager.java:37)
	at knot//com.thedeathlycow.novoatlas.fabric.MapImageLoader.method_14491(MapImageLoader.java:19)
	at knot//net.minecraft.class_4013.method_29490(class_4013.java:16)
	at java.base/java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:787)
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
	at knot//net.minecraft.class_4014.method_67574(class_4014.java:60)
	at knot//net.minecraft.class_1255.method_18859(class_1255.java:164)
	at knot//net.minecraft.class_4093.method_18859(class_4093.java:23)
	at knot//net.minecraft.class_1255.method_16075(class_1255.java:138)
	at knot//net.minecraft.class_1255.method_18857(class_1255.java:147)
	at knot//net.minecraft.class_525.method_64244(class_525.java:373)
	at knot//net.minecraft.class_525.method_64368(class_525.java:321)
	at knot//net.minecraft.class_525.method_31130(class_525.java:308)
	at knot//net.minecraft.class_526.method_19944(class_526.java:54)
	at knot//net.minecraft.class_4185.method_25306(class_4185.java:96)
	at knot//net.minecraft.class_4264.method_25348(class_4264.java:43)
	at knot//net.minecraft.class_339.method_25402(class_339.java:141)
	at knot//net.minecraft.class_4069.method_25402(class_4069.java:43)
	at knot//net.minecraft.class_312.mixinextras$bridge$method_25402$38(class_312.java)
	at knot//net.minecraft.class_312.wrapOperation$zoa000$fabric-screen-api-v1$invokeMouseClickedEvents(class_312.java:545)
	at knot//net.minecraft.class_312.method_1601(class_312.java:120)
	at knot//net.minecraft.class_312.method_22686(class_312.java:226)
	at knot//net.minecraft.class_1255.execute(class_1255.java:110)
	at knot//net.minecraft.class_312.method_22684(class_312.java:226)
	at knot//org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43)
	at knot//org.lwjgl.system.JNI.invokeV(Native Method)
	at knot//org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3509)
	at knot//com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:164)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1375)
	at knot//net.minecraft.class_310.method_1514(class_310.java:936)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:506)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:102)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
Caused by: java.lang.ClassNotFoundException: com.twelvemonkeys.imageio.spi.ImageReaderSpiBase
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	... 59 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at java.base/java.lang.ClassLoader.defineClass1(Native Method)
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027)
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:160)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:355)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:534)
	at java.base/java.lang.Class.forName(Class.java:513)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.nextProviderClass(ServiceLoader.java:1217)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1228)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1273)
	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1309)
	at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1393)
	at java.desktop/javax.imageio.spi.IIORegistry.registerApplicationClasspathSpis(IIORegistry.java:179)
	at java.desktop/javax.imageio.spi.IIORegistry.<init>(IIORegistry.java:107)
	at java.desktop/javax.imageio.spi.IIORegistry.getDefaultInstance(IIORegistry.java:128)
	at java.desktop/javax.imageio.ImageIO.<clinit>(ImageIO.java:64)
	at knot//com.thedeathlycow.novoatlas.registry.MultiFileTypeToIdConverter.builtinImages(MultiFileTypeToIdConverter.java:27)
	at knot//com.thedeathlycow.novoatlas.registry.MultiFileTypeToIdConverter.imageRegistry(MultiFileTypeToIdConverter.java:35)
	at knot//com.thedeathlycow.novoatlas.registry.ImageManager.reload(ImageManager.java:37)
	at knot//com.thedeathlycow.novoatlas.fabric.MapImageLoader.method_14491(MapImageLoader.java:19)
	at knot//net.minecraft.class_4013.method_29490(class_4013.java:16)
	at java.base/java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:787)
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
	at knot//net.minecraft.class_4014.method_67574(class_4014.java:60)
	at knot//net.minecraft.class_1255.method_18859(class_1255.java:164)
	at knot//net.minecraft.class_4093.method_18859(class_4093.java:23)
	at knot//net.minecraft.class_1255.method_16075(class_1255.java:138)
	at knot//net.minecraft.class_1255.method_18857(class_1255.java:147)
	at knot//net.minecraft.class_525.method_64244(class_525.java:373)
	at knot//net.minecraft.class_525.method_64368(class_525.java:321)
	at knot//net.minecraft.class_525.method_31130(class_525.java:308)
	at knot//net.minecraft.class_526.method_19944(class_526.java:54)
	at knot//net.minecraft.class_4185.method_25306(class_4185.java:96)
	at knot//net.minecraft.class_4264.method_25348(class_4264.java:43)
	at knot//net.minecraft.class_339.method_25402(class_339.java:141)
	at knot//net.minecraft.class_4069.method_25402(class_4069.java:43)
	at knot//net.minecraft.class_312.mixinextras$bridge$method_25402$38(class_312.java)
	at knot//net.minecraft.class_312.wrapOperation$zoa000$fabric-screen-api-v1$invokeMouseClickedEvents(class_312.java:545)
	at knot//net.minecraft.class_312.method_1601(class_312.java:120)
	at knot//net.minecraft.class_312.method_22686(class_312.java:226)
	at knot//net.minecraft.class_1255.execute(class_1255.java:110)
	at knot//net.minecraft.class_312.method_22684(class_312.java:226)
	at knot//org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43)
	at knot//org.lwjgl.system.JNI.invokeV(Native Method)
	at knot//org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3509)

-- Affected screen --
Details:
	Screen name: net.minecraft.class_526
Stacktrace:
	at knot//net.minecraft.class_312.method_1601(class_312.java:120)
	at knot//net.minecraft.class_312.method_22686(class_312.java:226)
	at knot//net.minecraft.class_1255.execute(class_1255.java:110)
	at knot//net.minecraft.class_312.method_22684(class_312.java:226)
	at knot//org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43)
	at knot//org.lwjgl.system.JNI.invokeV(Native Method)
	at knot//org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3509)
	at knot//com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:164)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1375)
	at knot//net.minecraft.class_310.method_1514(class_310.java:936)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:506)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:102)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)

-- Mouse --
Details:
	Mouse location: Scaled: (211.765641, 350.355469). Absolute: (422.941406, 700.710938)
	Screen size: Scaled: (359, 399). Absolute: (1434, 1596). Scale factor of 4.000000
	Button: 0
Stacktrace:
	at knot//net.minecraft.class_312.method_1601(class_312.java:120)
	at knot//net.minecraft.class_312.method_22686(class_312.java:226)
	at knot//net.minecraft.class_1255.execute(class_1255.java:110)
	at knot//net.minecraft.class_312.method_22684(class_312.java:226)
	at knot//org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43)
	at knot//org.lwjgl.system.JNI.invokeV(Native Method)
	at knot//org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3509)
	at knot//com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:164)
	at knot//net.minecraft.class_310.method_1523(class_310.java:1375)
	at knot//net.minecraft.class_310.method_1514(class_310.java:936)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:506)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:102)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)

-- Uptime --
Details:
	JVM uptime: 18.692s
	Wall uptime: 9.500s
	High-res time: 8.962s
	Client ticks: 134 ticks / 6.700s
Stacktrace:
	at knot//net.minecraft.class_310.method_1587(class_310.java:2382)
	at knot//net.minecraft.class_310.method_54580(class_310.java:1005)
	at knot//net.minecraft.class_310.method_1514(class_310.java:957)
	at knot//net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:506)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:102)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, fabric, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-tag-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, novoatlas

-- System Details --
Details:
	Minecraft Version: 1.21.5
	Minecraft Version ID: 1.21.5
	Operating System: Mac OS X (aarch64) version 14.6.1
	Java Version: 21.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 208527056 bytes (198 MiB) / 1161822208 bytes (1108 MiB) up to 6442450944 bytes (6144 MiB)
	CPUs: 8
	Processor Vendor: Apple Inc.
	Processor Name: Apple M2
	Identifier: Apple Inc. Family 0xda33d83d Model 0 Stepping 0
	Microarchitecture: ARM64 SoC: Avalanche + Blizzard
	Frequency (GHz): 3.50
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 8
	Graphics card #0 name: Apple M2
	Graphics card #0 vendor: Apple (0x106b)
	Graphics card #0 VRAM (MiB): 0.00
	Graphics card #0 deviceId: unknown
	Graphics card #0 versionInfo: unknown
	Memory slot #0 capacity (MiB): 0.00
	Memory slot #0 clockSpeed (GHz): 0.00
	Memory slot #0 type: unknown
	Virtual memory max (MiB): 14336.00
	Virtual memory used (MiB): 12358.70
	Swap memory total (MiB): 6144.00
	Swap memory used (MiB): 5385.06
	Space in storage for jna.tmpdir (MiB): <path not set>
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): <path not set>
	Space in storage for io.netty.native.workdir (MiB): <path not set>
	Space in storage for java.io.tmpdir (MiB): available: 5591.35, total: 233752.44
	Space in storage for workdir (MiB): available: 5591.35, total: 233752.44
	JVM Flags: 2 total; -Xms1024m -Xmx6144m
	Fabric Mods: 
		fabric-api: Fabric API 0.128.1+1.21.5
			fabric-api-base: Fabric API Base 0.4.62+73a52b4b49
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.96+86c3a9f149
			fabric-biome-api-v1: Fabric Biome API (v1) 16.0.7+2dd063df49
			fabric-block-api-v1: Fabric Block API (v1) 1.1.0+ed91556f49
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.26+aa6d566c49
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 2.0.16+86c3a9f149
			fabric-client-tags-api-v1: Fabric Client Tags 1.1.37+86c3a9f149
			fabric-command-api-v1: Fabric Command API (v1) 1.2.70+f71b366f49
			fabric-command-api-v2: Fabric Command API (v2) 2.2.49+73a52b4b49
			fabric-commands-v0: Fabric Commands (v0) 0.2.87+df3654b349
			fabric-content-registries-v0: Fabric Content Registries (v0) 10.0.14+3e6c1f7d49
			fabric-convention-tags-v1: Fabric Convention Tags 2.1.33+7f945d5b49
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.15.2+6d9989f349
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.3.12+86c3a9f149
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.8.5+6d9989f349
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 22.5.2+a0fab30c49
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.17+3e6c1f7d49
			fabric-entity-events-v1: Fabric Entity Events (v1) 2.1.0+3ce7866349
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 4.0.15+64e3057949
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.70+c327076a49
			fabric-item-api-v1: Fabric Item API (v1) 11.4.1+e46fd76a49
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.2.9+3459fc6149
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.63+ecf51cdc49
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.61+df3654b349
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.6.0+230071a049
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.48+3f89f5a549
			fabric-loot-api-v3: Fabric Loot API (v3) 1.0.36+86c3a9f149
			fabric-message-api-v1: Fabric Message API (v1) 6.1.0+fe971bba49
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 5.2.3+5281b90749
			fabric-networking-api-v1: Fabric Networking API (v1) 4.5.0+775be32c49
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 21.1.2+b8d6ba7049
			fabric-particles-v1: Fabric Particles (v1) 4.1.2+112e550e49
			fabric-recipe-api-v1: Fabric Recipe API (v1) 8.1.8+3235ab3249
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 6.1.22+cd64004349
			fabric-renderer-api-v1: Fabric Renderer API (v1) 6.1.2+bdf4842249
			fabric-renderer-indigo: Fabric Renderer - Indigo 3.1.2+5281b90749
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.64+73761d2e49
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.27+86c3a9f149
			fabric-rendering-v1: Fabric Rendering (v1) 11.3.0+a272b33849
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 5.0.21+73a52b4b49
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 3.1.7+847e5f5c49
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.46+86c3a9f149
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.129+c327076a49
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.38+86c3a9f149
			fabric-tag-api-v1: Fabric Tag API (v1) 1.0.17+ecf51cdc49
			fabric-transfer-api-v1: Fabric Transfer API (v1) 5.4.24+7b20cbb049
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.3.17+f17a180c49
		fabricloader: Fabric Loader 0.17.0
			mixinextras: MixinExtras 0.5.0
		java: OpenJDK 64-Bit Server VM 21
		minecraft: Minecraft 1.21.5
		novoatlas: NovoAtlas 1.2.0
			com_twelvemonkeys_imageio_imageio-webp: imageio-webp 3.12.0
	Launched Version: 1.21.5
	Launcher name: PrismLauncher
	Backend library: LWJGL version 3.3.3-snapshot
	Backend API: Apple M2 GL version 4.1 Metal - 88.1, Apple
	Window size: 1434x1596
	GFLW Platform: cocoa
	Render Extensions: 
	GL debug messages: <debugging unavailable>
	Is Modded: Definitely; Client brand changed to 'fabric'
	Universe: 400921fb54442d18
	Type: Client (map_client.txt)
	GPU Warnings: version: Metal
	Graphics mode: fancy
	Render Distance: 10/10 chunks
	Resource Packs: vanilla, fabric, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-tag-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, novoatlas
	Current Language: en_us
	Locale: en_IT
	System encoding: UTF-8
	File encoding: UTF-8
	CPU: 8x Apple M2
```

It's impossible to open already saved world too

```
java.util.concurrent.ExecutionException: java.lang.NoClassDefFoundError: com/twelvemonkeys/imageio/spi/ImageReaderSpiBase
	at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396) ~[?:?]
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073) ~[?:?]
	at knot/net.minecraft.class_7196.method_45694(class_7196.java:185) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_7196.method_54610(class_7196.java:138) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_7196.method_57780(class_7196.java:324) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_7196.method_57781(class_7196.java:314) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_7196.method_57782(class_7196.java:279) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_7196.method_57784(class_7196.java:245) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_528$class_4272.method_20164(class_528.java:432) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_528$class_4272.method_25402(class_528.java:409) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_4069.method_25402(class_4069.java:43) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_9017.method_25402(class_9017.java:57) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_4069.method_25402(class_4069.java:43) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_312.mixinextras$bridge$method_25402$42(class_312.java) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_312.wrapOperation$zpd000$fabric-screen-api-v1$invokeMouseClickedEvents(class_312.java:1045) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_312.method_1601(class_312.java:120) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_312.method_22686(class_312.java:226) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.execute(class_1255.java:110) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_312.method_22684(class_312.java:226) ~[client-intermediary.jar:?]
	at knot/org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:43) [lwjgl-glfw-3.3.3.jar:?]
	at knot/org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.3.3.jar:?]
	at knot/org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3509) [lwjgl-glfw-3.3.3.jar:?]
	at knot/com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:164) [client-intermediary.jar:?]
	at knot/net.minecraft.class_310.method_1523(class_310.java:1375) [client-intermediary.jar:?]
	at knot/net.minecraft.class_310.method_1514(class_310.java:936) [client-intermediary.jar:?]
	at knot/net.minecraft.client.main.Main.main(Main.java:265) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:506) [fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72) [fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.17.0.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:102) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129) [NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70) [NewLaunch.jar:?]
Caused by: java.lang.NoClassDefFoundError: com/twelvemonkeys/imageio/spi/ImageReaderSpiBase
	at java.base/java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027) ~[?:?]
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:160) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:355) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.17.0.jar:?]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
	at java.base/java.lang.Class.forName0(Native Method) ~[?:?]
	at java.base/java.lang.Class.forName(Class.java:534) ~[?:?]
	at java.base/java.lang.Class.forName(Class.java:513) ~[?:?]
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.nextProviderClass(ServiceLoader.java:1217) ~[?:?]
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1228) ~[?:?]
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1273) ~[?:?]
	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1309) ~[?:?]
	at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1393) ~[?:?]
	at java.desktop/javax.imageio.spi.IIORegistry.registerApplicationClasspathSpis(IIORegistry.java:179) ~[?:?]
	at java.desktop/javax.imageio.spi.IIORegistry.<init>(IIORegistry.java:107) ~[?:?]
	at java.desktop/javax.imageio.spi.IIORegistry.getDefaultInstance(IIORegistry.java:128) ~[?:?]
	at java.desktop/javax.imageio.ImageIO.<clinit>(ImageIO.java:64) ~[?:?]
	at knot/com.thedeathlycow.novoatlas.registry.MultiFileTypeToIdConverter.builtinImages(MultiFileTypeToIdConverter.java:27) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/com.thedeathlycow.novoatlas.registry.MultiFileTypeToIdConverter.imageRegistry(MultiFileTypeToIdConverter.java:35) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/com.thedeathlycow.novoatlas.registry.ImageManager.reload(ImageManager.java:37) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/com.thedeathlycow.novoatlas.fabric.MapImageLoader.method_14491(MapImageLoader.java:19) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/net.minecraft.class_4013.method_29490(class_4013.java:16) ~[client-intermediary.jar:?]
	at java.base/java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:787) ~[?:?]
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	at knot/net.minecraft.class_4014.method_67574(class_4014.java:60) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_18859(class_1255.java:164) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_4093.method_18859(class_4093.java:23) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_16075(class_1255.java:138) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_18857(class_1255.java:147) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_7196.method_45694(class_7196.java:184) ~[client-intermediary.jar:?]
	... 29 more
Caused by: java.lang.ClassNotFoundException: com.twelvemonkeys.imageio.spi.ImageReaderSpiBase
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.17.0.jar:?]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
	at java.base/java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027) ~[?:?]
	at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.defineClassFwd(KnotClassLoader.java:160) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:355) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.17.0.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.17.0.jar:?]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
	at java.base/java.lang.Class.forName0(Native Method) ~[?:?]
	at java.base/java.lang.Class.forName(Class.java:534) ~[?:?]
	at java.base/java.lang.Class.forName(Class.java:513) ~[?:?]
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.nextProviderClass(ServiceLoader.java:1217) ~[?:?]
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1228) ~[?:?]
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1273) ~[?:?]
	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1309) ~[?:?]
	at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1393) ~[?:?]
	at java.desktop/javax.imageio.spi.IIORegistry.registerApplicationClasspathSpis(IIORegistry.java:179) ~[?:?]
	at java.desktop/javax.imageio.spi.IIORegistry.<init>(IIORegistry.java:107) ~[?:?]
	at java.desktop/javax.imageio.spi.IIORegistry.getDefaultInstance(IIORegistry.java:128) ~[?:?]
	at java.desktop/javax.imageio.ImageIO.<clinit>(ImageIO.java:64) ~[?:?]
	at knot/com.thedeathlycow.novoatlas.registry.MultiFileTypeToIdConverter.builtinImages(MultiFileTypeToIdConverter.java:27) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/com.thedeathlycow.novoatlas.registry.MultiFileTypeToIdConverter.imageRegistry(MultiFileTypeToIdConverter.java:35) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/com.thedeathlycow.novoatlas.registry.ImageManager.reload(ImageManager.java:37) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/com.thedeathlycow.novoatlas.fabric.MapImageLoader.method_14491(MapImageLoader.java:19) ~[novoatlas-fabric-1.2.0.jar:?]
	at knot/net.minecraft.class_4013.method_29490(class_4013.java:16) ~[client-intermediary.jar:?]
	at java.base/java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:787) ~[?:?]
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	at knot/net.minecraft.class_4014.method_67574(class_4014.java:60) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_18859(class_1255.java:164) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_4093.method_18859(class_4093.java:23) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_16075(class_1255.java:138) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_1255.method_18857(class_1255.java:147) ~[client-intermediary.jar:?]
	at knot/net.minecraft.class_7196.method_45694(class_7196.java:184) ~[client-intermediary.jar:?]
	... 29 more
```

--- a/changelogs/1.2.1.md
+++ b/changelogs/1.2.1.md
@@ -1,0 +1,1 @@
+* Fixed crash on world creation due to missing class

--- a/fabric/build.gradle
+++ b/fabric/build.gradle
@@ -38,7 +38,21 @@ dependencies {
     common(project(path: ':common', configuration: 'namedElements')) { transitive false }
     shadowBundle project(path: ':common', configuration: 'transformProductionFabric')
 
-    implementation include ("com.twelvemonkeys.imageio:imageio-webp:${project.twelvemonkeys_imageio_version}")
+    List.of(
+            "imageio-webp",
+            "imageio-metadata",
+            "imageio-core"
+    ).forEach {
+        implementation include ("com.twelvemonkeys.imageio:$it:${project.twelvemonkeys_imageio_version}")
+    }
+
+    List.of(
+            "common-image",
+            "common-io",
+            "common-lang"
+    ).forEach {
+        implementation include ("com.twelvemonkeys.common:$it:${project.twelvemonkeys_imageio_version}")
+    }
 }
 
 processResources {

--- a/gradle.properties
+++ b/gradle.properties
@@ -3,7 +3,7 @@ org.gradle.jvmargs=-Xmx2G
 org.gradle.parallel=true
 
 # Mod properties
-mod_version = 1.2.0
+mod_version = 1.2.1
 maven_group = com.thedeathlycow.novoatlas
 archives_name = novoatlas
 enabled_platforms = fabric,neoforge

--- a/neoforge/build.gradle
+++ b/neoforge/build.gradle
@@ -42,7 +42,21 @@ dependencies {
     common(project(path: ':common', configuration: 'namedElements')) { transitive false }
     shadowBundle project(path: ':common', configuration: 'transformProductionNeoForge')
 
-    forgeRuntimeLibrary(implementation include ("com.twelvemonkeys.imageio:imageio-webp:${project.twelvemonkeys_imageio_version}"))
+    List.of(
+            "imageio-webp",
+            "imageio-metadata",
+            "imageio-core"
+    ).forEach {
+        forgeRuntimeLibrary(implementation include ("com.twelvemonkeys.imageio:$it:${project.twelvemonkeys_imageio_version}"))
+    }
+
+    List.of(
+            "common-image",
+            "common-io",
+            "common-lang"
+    ).forEach {
+        forgeRuntimeLibrary(implementation include ("com.twelvemonkeys.common:$it:${project.twelvemonkeys_imageio_version}"))
+    }
 }
 
 processResources {

